### PR TITLE
Build binary with Docker

### PR DESCRIPTION
**Pull request by johnwu96822**
To build `make docker.build`, must supply BUILDER_IMAGE value.

- [x] @abdulchaudhrycoupa 
- [x] @supriya

### CODE REVIEW

**Review by abdulchaudhrycoupa — APPROVED** [review]

**Comment by johnwu96822** [comment]
```
go test ./...
?   	github.com/ory/oathkeeper	[no test files]
ok  	github.com/ory/oathkeeper/api	(cached)
ok  	github.com/ory/oathkeeper/cmd	(cached)
?   	github.com/ory/oathkeeper/cmd/server	[no test files]
ok  	github.com/ory/oathkeeper/credentials	(cached)
ok  	github.com/ory/oathkeeper/driver	(cached)
ok  	github.com/ory/oathkeeper/driver/configuration	(cached)
ok  	github.com/ory/oathkeeper/helper	(cached)
?   	github.com/ory/oathkeeper/internal	[no test files]
?   	github.com/ory/oathkeeper/internal/cloudstorage	[no test files]
?   	github.com/ory/oathkeeper/internal/httpclient/client	[no test files]
?   	github.com/ory/oathkeeper/internal/httpclient/client/api	[no test files]
?   	github.com/ory/oathkeeper/internal/httpclient/client/health	[no test files]
?   	github.com/ory/oathkeeper/internal/httpclient/client/version	[no test files]
?   	github.com/ory/oathkeeper/internal/httpclient/models	[no test files]
ok  	github.com/ory/oathkeeper/metrics	(cached)
?   	github.com/ory/oathkeeper/packrd	[no test files]
?   	github.com/ory/oathkeeper/pipeline	[no test files]
ok  	github.com/ory/oathkeeper/pipeline/authn	(cached)
ok  	github.com/ory/oathkeeper/pipeline/authz	(cached)
ok  	github.com/ory/oathkeeper/pipeline/errors	(cached)
ok  	github.com/ory/oathkeeper/pipeline/mutate	(cached)
ok  	github.com/ory/oathkeeper/proxy	(cached)
ok  	github.com/ory/oathkeeper/rule	(cached)
?   	github.com/ory/oathkeeper/test/e2e/okapi	[no test files]
?   	github.com/ory/oathkeeper/test/e2e/okclient	[no test files]
ok  	github.com/ory/oathkeeper/x	(cached)
```

File: driver/registry_memory_test.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 func TestRegistryMemoryAvailablePipelineAuthorizers(t *testing.T) {
 	r := NewRegistryMemory()
 	got := r.AvailablePipelineAuthorizers()
-	assert.ElementsMatch(t, got, []string{"allow", "deny", "keto_engine_acp_ory", "remote", "remote_json"})
+	assert.ElementsMatch(t, got, []string{"allow", "deny", "keto_engine_acp_ory", "warden", "remote", "remote_json"})
```
Comment (supriya) [inline driver/registry_memory_test.go:12]: I guess this is an unrelated change.

**Comment by supriya** [comment]
lgtm

File: driver/registry_memory_test.go (hunk 1 above)
Comment (johnwu96822) [reply driver/registry_memory_test.go:12]: This is a test fix

**Comment by johnwu96822** [comment]
[lockit v4.9.3] merged via Lockit CLI on 2020-12-08 15:21:15.848918 -0800 PST m=+4.202136041
